environment_setup/setup_grsim_arch.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# UBC Thunderbots Arch Linux grSim Setup
+#
+# This script will install all the required libraries and dependencies to run
+# grSim. This only includes the simulator (grSim), and not anything related to
+# our AI. See the setup_software.sh script for that.
```

Comment:
Could we add a note that this is _not_ an officially supported script?

Reply:
added